deploy/template_samples/clustered_hana.json

```
@@ -5,6 +5,11 @@
       "is_existing": "false",
       "name": "test-rg"
     },
+    "ppg":
+    {
```

nit: the format in the json should be consistent.
```suggestion
    "ppg": {
```